style/javascript/sample.js

```diff
@@ -0,0 +1,7 @@
+object = {spacing: false}
```

Comment:
I've been skipping the spacing on the latest project because it can look/feel weird when nested inline. Most of the time it will look like this:

```
{
  spacing: false
}
```

So I don't think it's that bad. I don't feel strongly about this one, so if we prefer the space I'm fine with that too.